packages/test/src/data-converters/data-converter.ts

```diff
@@ -0,0 +1,6 @@
+import { DefaultDataConverter } from '@temporalio/common';
+import root, { foo } from '../../protos/root';
+
+export const messageInstance = foo.bar.ProtoActivityInput.create({ name: 'Proto', age: 1 });
```

Comment:
Should this be here?

Reply:
It's used in some of the tests. Do you think it's better to create it in the tests?